rasa/core/policies/rule_policy.py

```diff
@@ -489,3 +492,16 @@ def _default_predictions(self, domain: Domain) -> List[float]:
                 domain.index_for_action(self._fallback_action_name)
             ] = self._core_fallback_threshold
         return result
+
+    def _metadata(self) -> Dict[Text, Any]:
+        return {
+            "priority": self.priority,
+            "lookup": self.lookup,
```

Comment:
```suggestion
            **super()._metadata(),
```

Reply:
`RulePolicy` doesn't have `max_history` so that would break instantiation